Core/WumpusCore/Controller/Controller.cs

```diff
@@ -57,6 +91,26 @@ public Trivia.AskableQuestion GetTriviaQuestion()
         {
             return trivia.GetQuestion();
         }
-    }
 
+        public void StartGame()
+        {
+            ValidateScene(new []{StartScreen},InRoom);
+            this.state = InRoom;
+        }
+
+        /// <summary>
+        /// Throws an invalid operations exception if the current state is not in the valid states. Meant to be used as validation for methods to prevent UI from getting any funny ideas
```

Comment:
Please say what it's doing before saying the exception it'll throw if you do it wrong